pages/user guides/tidal-vms.md

```diff
@@ -1,35 +1,60 @@
 ---
-title: Pre-Configured Virtual Machines
+title: Tidal Tools Virtual Machine Image
 keywords: vm, virtual, machine, image, ova, ovf, aws, ami, tidal, tools
-last_updated: February, 2022
-summary: "Virtual machines pre-configured with Tidal Migrations software"
+last_updated: April, 2022
+summary: "Tidal Tools virtual image for your migration discovery"
 sidebar: main_sidebar
-permalink: pre-configured-vms.html
+permalink: tidal-discovery-vm.html
 ---
 
-To help you get your Tidal Migrations discovery phase up and running, we've created a pre-configured virtual machine image. This VM has the latest Tidal Migrations software and dependencies installed for you, so you can focus on getting things done.
-This virtual machine can also run as a server that is isolated from the public internet. To run Tidal Migrations software offline, checkout [this guide](https://guides.tidalmg.com/tidal-offline.html).
+To help you get your Tidal Migrations discovery done easily and quickly you can use the Tidal Tools virtual machine image. This VM has the latest Tidal Migrations software and dependencies installed for you, so you can focus on getting things done.
+It can also perform discovery without any network connectivity to the public internet. To run a discovery in an 'offline' mode checkout [this guide](https://guides.tidalmg.com/tidal-offline.html).
 
-## Ubuntu 18.04
+## Get Started
```

Comment:
I think the initial idea behind having the OSes as the headers was that we'd eventually have multiple, but I think it makes sense to present it like this while we just have the one.